dev/com.ibm.websphere.appserver.javax.annotation-1.3/com.ibm.websphere.appserver.javax.annotation-1.3.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.javax.annotation-1.3
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.javaee.annotation.1.3; location:="dev/api/spec/,lib/"
+kind=noship
```

Comment:
This will need to be `kind=beta` and `edition=core` because there are beta features (such as jaxrs-2.1) that need to pull in this feature